app/views/admin/applicants/index.html.haml

```diff
@@ -1,6 +1,13 @@
 - content_for(:page_header) { t('admin.applicants.index.header') }
 
 %p= t('admin.applicants.index.helper_text')
+-# I need to some how put a form here, with a submit button that gets and replaces @applicants
+
+= form_with url: admin_applicants_url, method: "get" do |f|
+  = f.label :q, "Search for:"
+  = f.text_field :q
+  = f.submit "Search"

```

Comment:
@MatthewSchultz how would I update ```@applicants``` based on this's response?

Reply:
So what this will do is send a get request to `/admin/applicants` with the **params** variable setup to include **q** (should be renamed to query probably).

That, as we already know, maps to the index action of the admin applicants controller. So, in that index action, we look for the query variable, use it to search applicants, and then set `@applicants` to the result. That stuff then gets rendered appropriately in the view.

Reply:
Reading the logs, and some printing, I see the query working and returning 1 item/etc. I see the template get rerendered. But on my browser I still see all records in the table.

I'm not entirely sure what's missing :S